gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/javaversions/BaselineJavaVersionsExtension.java

```diff
@@ -77,6 +82,14 @@ public final void setLibraryTarget(String value) {
         libraryTarget.set(version.javaLanguageVersion());
     }
 
+    public final Property<JavaLanguageVersion> getDaemonTarget() {
```

Comment:
I don't think this should live here - imo it should live in gradle-jdks in it's own extension as it's not needed in baseline at all. It should also be possible to use gradle-jdks without baseline-java-versions - it's not at the moment, but with the conclusion of the new work we should be able to remove the dependency.

Reply:
I was thinking that for consistency it would be easier to have all the java versions in the same extension

Reply:
I can see how that would be nice but I think it still logically makes sense as part of gradle-jdks. We've done similar things in the past (palantir-java-format config in the spotless part of baseline) and ended up having to unwind it later, which was kinda painful.